core/src/components/datetime/datetime.tsx

```diff
@@ -308,58 +308,77 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() firstDayOfWeek = 0;
 
+  /**
+   * If `true`, multiple dates can be selected at once. Only
+   * applies to `presentation="date"` and `preferWheel="false"`.
+   */
+  @Prop() multiple = false;
+
   /**
    * The value of the datetime as a valid ISO 8601 datetime string.
+   * Will be an array of strings if `multiple="true"`.
```

Comment:
Hmm... more the `ionChange` piece, though it also applies to the `value` you can access via e.g. `datetimeEl.value`. I think saying "should" instead of "will" gets closer to what I meant.